vllm/entrypoints/openai/api_server.py

```diff
@@ -955,8 +955,10 @@ def _listen_addr(a: str) -> str:
                 return '[' + a + ']'
             return a or "0.0.0.0"
 
-        logger.info("Starting vLLM API server on http://%s:%d",
-                    _listen_addr(sock_addr[0]), sock_addr[1])
+        is_ssl = args.ssl_keyfile and args.ssl_certfile
+        logger.info("Starting vLLM API server on http%s://%s:%d",
+                    is_ssl and "s" or "", _listen_addr(sock_addr[0]),
```

Comment:
```suggestion
                    "s" if is_ssl else "", _listen_addr(sock_addr[0]),
```

Make this a bit more readable

Reply:
I do like that better, thanks!